### DISCUSSION TITLE
colour picker is behind the sidebar

### DISCUSSION THREAD

**Discussion by mariomatasystems** [cables-gl/cables_docs#781] [! Welcome - readme first!]
Noticed the colourpicker is behind the sidebar
and looks like the spacing is a little funny

<img width="1314" alt="Screenshot 2024-06-04 at 10 30 27" src="https://github.com/cables-gl/cables_docs/assets/162353676/1935e7d3-e51e-4cc6-aa34-5d4535108bf9">

https://cables.gl/p/8-XQ5d

**Comment by pandrr** [comment]
dupe #770
